.github/stale.yml

```diff
@@ -0,0 +1,23 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 90
+
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
+
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - pinned
+  - security
+
+# Label to use when marking an issue as stale
+staleLabel: stale
+
+# Comment to post when marking an issue as stale. Set to `false` to disable
+markComment: >
+  This issue has been automatically marked as stale because it has not had
+  recent activity. It will be closed if no further activity occurs. Thank you
+  for your contributions. For more information, see [Stack's old issue close
+  policy](https://github.com/commercialhaskell/stack/blob/stable/doc/maintainers/team_process.md#old-issue-closing-policy).
```

Comment:
Either in the policy, or more likely here, we should probably emphasize that if somebody is affected by such a closed issue, it's enough to ask to reopen it even later.

Reply:
@nh2 Since you objected to this policy elsewhere: it'd be lovely to try to find a wording that you'd be comfortable with, as a compromise — if we do go through with this PR. I can try sketching something, but I'd like to build it off a consensus.

Reply:
Sometimes issues don't get closed even though they have been resolved, through oversight. Perhaps whoever is on triage could run through recently stale issues and identify which might have been fixed, so they can be marked as resolved.